pandas/core/groupby/generic.py

```diff
@@ -1687,8 +1687,10 @@ def _wrap_aggregated_output(
         -------
         DataFrame
         """
+        idx_name = output.get("idx_name")
```

Comment:
Just `idx_name  = output.pop("idx_name", None)`

Reply:
Ah thanks, did not think about that. Added the pop function later :)

Reply:
Done